html-forms/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>html-forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <div><label for="#email">Email</label></div>
+  <input type="text" name="subscribe" id="email"/>
+  <div><label for="#codingCool">Why is coding cool?</label></div>
+  <select name="options" id="codingCool">
+    <option value="Select an option">Select an option</option>
+    <option value="Because it's interesting">Because it's interesting</option>
+    <option value="Because it's fun">Because it's fun</option>
+    <option value="Because it's powerful">Because it's powerful</option>
+  </select>
+  <div>
+    <p></p>
```

Comment:
please remove empty elements if you're not using them!

Reply:
Okay. I only used them to create space.  Could I delete them and change that wrapped label into a <p>?

Reply:
into a p element